app/templates/views/roadmap.html

```diff
@@ -29,69 +29,42 @@ <h2 class="heading-medium" id="things-we-are-working-on">Things we’re working
 
   <ul class="govuk-list govuk-list--bullet">
 
-    <li>Let services upload their own logo for emails and letters</li>
     <li>Start to <a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">migrate our database from GOV.UK PaaS</a> to one hosted using <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/rds/">Amazon RDS</a></li>
-    <li>Migrate one of our applications from GOV.UK PaaS to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
-    <li>Make it quicker and easier to turn off <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.trial_mode')}}">trial mode restrictions</a></li>
-    <li>Let services choose their own two-factor authentication method</li>
-    <li>Let services <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.guidance_receive_text_messages') }}">receive text messages</a> without having to contact the Notify team first</li>
-    <li>Make it easier to find information and guidance on the Notify website</li>
-    <li>Design reusable content patterns for text messages</li>
-    <li>Update the version of the <a class="govuk-link govuk-link--no-visited-state" href="https://design-system.service.gov.uk/">GOV.UK Design System</a> that we use</li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrating our database and applications from GOV.UK PaaS</a> to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
```

Comment:
_database_ replicates the point above
```suggestion
    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrating our applications from GOV.UK PaaS</a> to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
```

Reply:
Taking Chris’s suggestion and making it consistent with the other verbs in the list.

```suggestion
    <li><a class="govuk-link govuk-link--no-visited-state" href="https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/">Migrate our applications from GOV.UK PaaS</a> to <a class="govuk-link govuk-link--no-visited-state" href="https://aws.amazon.com/ecs/">Amazon ECS</a></li>
```